build_depends.repos

```diff
@@ -29,7 +29,7 @@ repositories:
   core/autoware_lanelet2_extension:
     type: git
     url: https://github.com/autowarefoundation/autoware_lanelet2_extension.git
-    version: main
```

Comment:
The build error happens with the main branch.
The same version in the autoware.universe will be used with this change.
https://github.com/autowarefoundation/autoware_universe/blob/8f64d8eaee6bf73358b38309951b8263968be830/build_depends_humble.repos#L14